apps/src/netsim/NetSimTable.js

```diff
@@ -123,6 +149,19 @@ NetSimTable.prototype.readAll = function (callback) {
 };
 
 /**
+ * @param {number} rowID
+ * @param {!NodeStyleCallback} callback
+ */
+NetSimTable.prototype.readFromID = function (rowID, callback) {
```

Comment:
I believe this should be called updateFromID. The callback only needs to be a completion callback;  the callback function can invoke functions that interact with the cache.  

I be we probably want to make this @private because there is currently no reason why general code should invoke this directly.